zerver/openapi/zulip.yaml

```diff
@@ -9053,31 +9058,29 @@ paths:
               to optimize network performance. This is an important optimization
               in organizations with 10,000s of users.
               <br />
-              New in Zulip 3.0 (feature level 18).
+              **Changes**: New in Zulip 3.0 (feature level 18).
 
             - `stream_typing_notifications`: Boolean for whether the client
               supports stream typing notifications.
               <br />
-              New in Zulip 4.0 (feature level 58). This capability is
+              **Changes**: New in Zulip 4.0 (feature level 58). This capability is
               for backwards-compatibility; it will be required in a
               future server release.
 
             - `user_settings_object`: Boolean for whether the client supports the modern
-              `user_settings` event type. If False, the server will additionally send the
-              legacy `update_display_settings` and `update_global_notifications` event
-              types for backwards-compatibility with clients that predate this API migration.
+              `user_settings` event type. If false, the server will additionally send the
+              legacy `update_global_notifications` and `update_display_settings` event
+              types.
               <br />
-              <br />
-              Because the feature level 89 API changes were merged together, clients can
-              safely make a request with this client capability and requesting all of the
-              `user_settings`, `update_display_settings`, and
-              `update_global_notifications` event types, and get exactly one copy of
-              settings data on any server version. (And then use the `zulip_feature_level`
-              in the `/register` response or the presence/absence of a `user_settings` key
-              to determine where to look).
-              <br />
-              New in Zulip 5.0 (feature level 89). This capability is for
+              **Changes**: New in Zulip 5.0 (feature level 89). This capability is for
               backwards-compatibility; it will be removed in a future server release.
+              Because the feature level 89 API changes were merged together, clients can
+              safely make a request with this client capability and also request all three
+              event types (`user_settings`, `update_display_settings`,
+              `update_global_notifications`), and get exactly one copy of settings data on
+              any server version. Clients can then use the `zulip_feature_level` in the
+              `/register` response or the presence/absence of a `user_settings` key to
+              determine where to look for the data.
```

Comment:
When doing the audit, I spent a little time cleaning up the `client_capabilities` parameter description to the `/register` endpoint. [Link to current documentation](https://zulip.com/api/changelog#changes-in-zulip-31).

Mostly it was adding **Changes** to the notes, but I did rework the `user_settings_object` description significantly. Below is a screenshot of the updated documentation, which hopefully conveys the same information in a cleaner and clearer way.

<details>
<summary>Screenshot of updated `client_capabilities` parameter</summary>

![Screenshot from 2022-06-23 18-28-34](https://user-images.githubusercontent.com/63245456/175349587-e1426d4b-29ee-4288-9384-21a61c68b924.png)

</details>

Reply:
I agree, thanks!